### PR TITLE
Addition of a throttle message structure

### PR DESCRIPTION
**Pull request by PBechon**
Implement feature for #1 and for a feature request on the C# repository (that was not opened on the right repository) https://github.com/LRTNZ/KerbalSimpitRevamped/issues/3

The idea is to add a throttleMessage struct to more easily send throttle message and to add an example to show how to control throttle or rotation.

### CODE REVIEW

**Comment by LRTNZ** [comment]
Just to clarify before merging this in - this is all working fine?

File: src/PayloadStructs.h [hunk 1]
```diff
@@ -96,6 +96,12 @@ struct wheelMessage {
   byte mask;
 } __attribute__((packed));
 
+/** A throttle control message.
+    This struct contains information about wheel steering and throttle. */
```
Comment (LRTNZ) [inline src/PayloadStructs.h]: Wheel Steering? The struct only has throttle information in it?

File: examples/KerbalSimpitThrottleDemo/KerbalSimpitThrottleDemo.ino [hunk 2]
```diff
@@ -0,0 +1,61 @@
+/* KerbalSimpitThrottleDemo
+
+   A demonstration of how to use KerbalSimPit to send throttle 
+   or rotation command
+
+   Assume A0, A1, A2 are plugged on a potentiometer
```
Comment (LRTNZ) [inline examples/KerbalSimpitThrottleDemo/KerbalSimpitThrottleDemo.ino]: Could you please include a link to a well documented resource (Sparkfun, Arduino, Adafruit), that shows how to do this, to help people new to Arduinos and electronics in general.

File: examples/KerbalSimpitThrottleDemo/KerbalSimpitThrottleDemo.ino [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  // Send the message
+  mySimpit.send(THROTTLE_MESSAGE, throttle_msg);
+
+  rotationMessage rot_msg;
+  // Read the values of the potentiometers
+  int reading_pitch = analogRead(PITCH_PIN);
+  int reading_roll = analogRead(ROLL_PIN);
+  // Convert them in KerbalSimpit range (you can add deadzone here)
+  rot_msg.pitch = map(reading_pitch, 0, 1023, INT16_MIN, INT16_MAX);
+  rot_msg.roll = map(reading_roll, 0, 1023, INT16_MIN, INT16_MAX);
```
Comment (LRTNZ) [inline examples/KerbalSimpitThrottleDemo/KerbalSimpitThrottleDemo.ino]: Example of deadzone?

**Comment by PBechon** [comment]
Yes, I tested it on a Arduino uno with a single potentiometer that I sequetially plug in each of the 3 input pin used as I did not have enough potentiometers ;)

**Review by LRTNZ — CHANGES_REQUESTED** [review]

File: src/PayloadStructs.h (hunk 1 above)
Comment (PBechon) [reply src/PayloadStructs.h]: Fixed, it was a copy-pasted that did not apply here.

File: examples/KerbalSimpitThrottleDemo/KerbalSimpitThrottleDemo.ino (hunk 2 above)
Comment (PBechon) [reply examples/KerbalSimpitThrottleDemo/KerbalSimpitThrottleDemo.ino]: I added two links and improved the description

File: examples/KerbalSimpitThrottleDemo/KerbalSimpitThrottleDemo.ino (hunk 3 above)
Comment (PBechon) [reply examples/KerbalSimpitThrottleDemo/KerbalSimpitThrottleDemo.ino]: I did not find clean code for deadzone, I think it is better to keep the example simple enough. I removed the mention of deadzone.

Comment (LRTNZ) [reply examples/KerbalSimpitThrottleDemo/KerbalSimpitThrottleDemo.ino]: Sweet!

**Review by LRTNZ — APPROVED** [review]

**Pull request merged**